pkg/kubectl/resource/builder.go

```diff
@@ -274,12 +275,48 @@ func (b *Builder) SelectAllParam(selectAll bool) *Builder {
 	return b
 }
 
+func hasNamesArg(args []string) bool {
+	if len(args) > 1 {
+		return true
+	}
+	newArgs := []string{}
+	for _, s := range args {
+		newArgs = append(newArgs, strings.Split(s, ",")...)
```

Comment:
I'm confused, why do we need this for loop? I think we don't allow `name` in the form of resource1/name1,resource2/name2.